lisp/ess-inf.el

```diff
@@ -1043,6 +1043,11 @@ If TOGGLE-EOB is given, the value of
 ;;       (display-buffer buf nil (get-frame-for-buffer buf))
 ;;     (display-buffer buf nil t)))
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(defcustom ess-show-buffer-action
+  '((display-buffer-use-some-window display-buffer-pop-up-window))
```

Comment:
You have reverted the order of actions. People already started complaining of the changed behavior. I didn't notice that during the merge. What was the reason for the change?